pyproject.toml

```diff
@@ -30,6 +30,7 @@ toml = ">=0.10.2"
 pytest = "^7.1.1"
 pytest-mock = "^3.7.0"
 cfn-lint = "^0.75.1"
+exasol-toolbox = "^0.12.0"
```

Comment:
```suggestion
exasol-toolbox = ">=0.12.0,<1.0.0"
```

Reply:
When the version is less than 1.0.0 the `^` increments the second number, not the first to get the upper boundary.

Reply:
Good point

Reply:
But I removed toolbox dependency for now